src/scmdata/run.py

```diff
@@ -1000,7 +1002,10 @@ def _meta_column(self, col: str) -> pd.Series[Any]:
         return pd.Series(out, name=col, index=self._df.columns)
 
     def set_meta(
-        self, dimension: str, value: MetadataValue, **filter_kwargs: MetadataValue
+        self,
+        dimension: str,
+        value: Union[MetadataValue, Iterable[MetadataValue]],
+        **filter_kwargs: Union[MetadataValue, List[MetadataValue]],
```

Comment:
```suggestion
        **filter_kwargs: Union[MetadataValue, Iterable[MetadataValue]],
```

I believe that iterables can be used. if that isn't the case that would indicate a bug 😄